storage2/src/lib.rs

```diff
@@ -1 +1,15 @@
+// Required to ensure that Rust can infer a Send bound inside the TCT
```

Comment:
We don't need this in this crate, I think, because we're not using the TCT here.